staging/BUILD

```diff
@@ -210,6 +210,7 @@ filegroup(
         "//staging/src/k8s.io/code-generator/cmd/lister-gen:all-srcs",
         "//staging/src/k8s.io/code-generator/cmd/openapi-gen:all-srcs",
         "//staging/src/k8s.io/code-generator/cmd/set-gen:all-srcs",
+        "//staging/src/k8s.io/code-generator/pkg/util:all-srcs",
```

Comment:
@mikedanese this is ugly that every change in some deeply nested staging repo requires approval on staging/ level.